app/controllers/partners/profiles_controller.rb

```diff
@@ -90,7 +90,7 @@ def profile_params
         :enable_individual_requests,
         :enable_quantity_based_requests,
         documents: []
-      ).select { |_, v| v.present? }
+      )
```

Comment:
There was a reason this had to happen before... there was some error that was happening. I don't remember why. :cry:

Reply:
Yeah when I remove that I actually got the error from: https://github.com/rubyforgood/human-essentials/issues/3446

## reason that I removed
So the thing here is we check with `v.present?`. So if there is no value, we are not getting any values from params and not updating and causing this bug.

## possible error that you metioned
As for the error that you talked I think it is this one(but I am not very sure):
https://github.com/rubyforgood/human-essentials/issues/3446

if we don't include `v.present` it is updating files(previously) and getting the error.

I don't know whether I keep the current change or (change the select method)?